examples/pos-mainchain/ecosystem.config.js

```diff
@@ -0,0 +1,32 @@
+const os = require('os');
+const num = 10;
+
+const followers = new Array(num).fill(0).map((_, i) => ({
+	name: `follower_${i}`,
+	script: './bin/run',
+	args: 'start',
+	interpreter: 'node',
+	env: {
+		LISK_LOG_LEVEL: 'debug',
+		LISK_NETWORK: 'alphanet',
+		LISK_PORT: 7667 + i + 1,
+		LISK_SEED_PEERS: `127.0.0.1:7667`,
+		LISK_DATA_PATH: `${os.tmpdir()}/follower_${i}`,
```

Comment:
How is deletion of files from `/tmp/follower_x` being handled, when a user wants to start a new ecosystem from genesis block?

How about using a subfolder like `${os.tmpdir()}/lisk-ecosystem/follower_${i}` and also include the seed node's data there, so user can reset the whole ecosystem using `rm -rf /tmp/lisk-ecosystem/`?